scripts/infinity_grid.py

```diff
@@ -171,6 +172,7 @@ def try_init():
     registerMode("HighRes Upscaler", GridSettingMode(dry=True, type="text", apply=apply_field("hr_upscaler"), valid_list=lambda: list(map(lambda u: u.name, shared.sd_upscalers)) + list(shared.latent_upscale_modes.keys())))
     registerMode("Image CFG Scale", GridSettingMode(dry=True, type="decimal", min=0, max=500, apply=apply_field("image_cfg_scale")))
     registerMode("Use Result Index", GridSettingMode(dry=True, type="integer", min=0, max=500, apply=apply_field("inf_grid_use_result_index")))
+    registerMode("batch size", GridSettingMode(dry=True,type="integer", apply=apply_field("batch_size")))
```

Comment:
This should have a name more like `[Grid Gen] Max AutoBatch Size` and tracked with a variable that will not be confused with the auto webui standard batch size